c3d/c3d.py

```diff
@@ -1478,10 +1673,8 @@ def read_frames(self, copy=True):
 
         if is_float:
             point_word_bytes = 4
-            point_dtype = self._dtypes.uint32
         else:
             point_word_bytes = 2
-            point_dtype = self._dtypes.int16
         points = np.zeros((self.point_used, 5), np.float32)
 
         # TODO: handle ANALOG:BITS parameter here!
```

Comment:
I sthat comment still relevant?

Reply:
Created Param.analog_format, Param.analog_format_unsigned and Param.analog_resolution properties to provide access to the standardized parameters (ANALOG:BITS, ANALOG:FORMAT).  

Manager.get_analog_transform_parameters() now utilize the Param.analog_format_unsigned property to determine how to read the ANALOG:OFFSET parameter, Reader.read_frames() parsing was updated to utilize the property as well. read_frames() also no longer check the BITS value as this is only defining the bit resolution for the measurement rather then how they are stored. It was only checked for UNSIGNED format which never occurs in any of the current test files, and it was probably only meant as a temporary solution triggered if there was a file on UNSIGNED form with an odd BITS parameter (hence the comment). It could plausible also exist a BITS parameter with non-standard interpretation (i.e. case where it represent the BITS for each value stored in the file). If so handle it if such a file can be found. 

It should also be noted that I have been unable to test the parsing of ANALOG input properly, current implementation has focused on trying to follow the standard but may contain misinterpretations or bugs due to lack of testing capabilities. POINT data parsing has been slightly easier to test but could in some cases suffer from similar problems.

Reply:
Thanks for clarification. Let's just leave the comment there and regarding analog channels: Let's see if people come across any bugs. The basics seem to work.